libcontainer/logs/logs_linux_test.go

```diff
@@ -13,6 +13,7 @@ import (
 
 func TestLoggingToFile(t *testing.T) {
 	l := runLogForwarding(t)
+	defer l.cleanup()
```

Comment:
Can we have a separate commit for `libcontainer/logs/logs_linux_test.go` so once the time comes we can just revert it?

Reply:
done